src/index.ts

```diff
@@ -74,6 +74,11 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
       const result = await runPublish({
         script: publishScript,
         githubToken,
+        options: {
+          createGithubReleases: JSON.parse(
+            getOptionalInput("createGithubReleases") ?? "true"
+          ),
```

Comment:
I've just learned about this helper function that can probably be used here:
```suggestion
          createGithubReleases: core.getBooleanInput("createGithubReleases"),
```

I believe that it takes the default declared in the `.yml` file into consideration but it would be great if we could confirm that.

Reply:
I don't think this is safe to do. That function (right from its type-def returns a boolean), but this is optional.

https://github.com/actions/toolkit/blob/daf8bb00606d37ee2431d9b1596b88513dcf9c59/packages/core/src/core.ts#L171-L182

In case people don't provide this (which is pretty much for all current users), it would throw error.

Reply:
I think that if it's not safe to do then we've made a horrible mistake merging this: https://github.com/changesets/action/pull/131/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R17 . While merging I didn't notice that it's using this `getBooleanInput` and I didn't check it back then. If this would throw then we'd already have some angry users in the issues about broken workflow runs :sweat_smile:

I think that the defaults might be provided by the workflow runner using environment variables:
https://github.com/actions/toolkit/blob/daf8bb00606d37ee2431d9b1596b88513dcf9c59/packages/core/src/core.ts#L129-L130

I'll try to recheck this later in my private playground repo.

Reply:
Could also be because people are using `changesets/action@v1` like in the `README.md` file 😅
A bit unrelated, do you know why this action doesn't show up in Github action marketplace, i.e. why don't we have this in our repo page?

![image](https://user-images.githubusercontent.com/13574879/149135222-47e4c8ab-4f01-4ee6-b5c3-08ca0c5bbd7b.png)

Reply:
> Could also be because people are using changesets/action@v1 like in the README.md file 

I've rechecked in my private github actions playground and it seems like a call to `core.getBooleanInput("createGithubReleases")` is enough :)

> A bit unrelated, do you know why this action doesn't show up in Github action marketplace, i.e. why don't we have this in our repo page?

Not sure, I'm not sure what's the requirement for this.